Makefile

```diff
@@ -188,6 +188,11 @@ ifeq ($(USE_OPERATOR_TUNING), 1)
 	CFLAGS += -DMXNET_USE_OPERATOR_TUNING=1
 endif
 
+ifeq ($(USE_INT64_TENSOR_SIZE), 1)
+   CFLAGS += -DMSHADOW_INT64_TENSOR_SIZE=1
```

Comment:
Since USE_INT64_TENSOR_SIZE has been set by default to 0 this is a breaking change. Are we going to add an exception for semver for the next minor release ?

Reply:
This does not break API but will impact cases for large tensors. How should we handle this cases?

Reply:
Ya probably doesn't fall under semver. We should still let users like dgl know about the change and how they can build for large tensor support.

Reply:
Will the MXNet pip packages be built with large tensor support disabled after this PR is merged? If so, for users who want large tensor support, do they have to build MXNet from source?

Reply:
Yes, users who need large tensor support will have to build MXNet from source.

Reply:
I think we need to document this. Maybe on the build from source page?

Reply:
It's documented in the config.mk file: https://github.com/apache/incubator-mxnet/pull/14570/files#diff-ea86416d0f764a60d29337663f7a1767R220